.github/README.md

```diff
@@ -0,0 +1,55 @@
+# [Oppia-Android](https://www.oppia.org)
+
+Oppia-Android is a mobile application version of [Oppia](https://www.oppia.org). It is an android application that enables anyone to easily learn interactive activities (called 'explorations'). These activities simulate a one-on-one conversation with a tutor, making it possible for students to learn by doing while getting feedback.
+
+In addition to that users can create their profile, pratice a set of skills, play with exporations, review previously finished skills and can also track the progress.
+
+Oppia-Android app is written using Kotlin and python, and is built on top of Google App Engine.
+
+  * [Oppia.org community site](https://www.oppia.org)
+  * [User Documentation](https://oppia.github.io/)
+  * [Contributors' wiki](https://github.com/oppia/oppia-android/wiki)
+  * [Developer mailing list](http://groups.google.com/group/oppia-android-dev)
+  * [File an issue](https://github.com/oppia/oppia-android/issues/new/)
+
+<p align="center">
+  <img width="361" alt="Screenshot 2019-12-18 at 4 37 57 PM" src="https://user-images.githubusercontent.com/9396084/71081269-ddf6f280-21b4-11ea-9a49-fcbd65b4ced7.png">
+</p>
+
+
+## Installation
+
+Please refer to the [developer wiki](https://github.com/oppia/oppia-android/wiki) for full installation instructions. This is just a short summary for developers who would like to contribute:
```

Comment:
Please remove the word short before summary.
"This is just a short summary for developers who would like to contribute:"

Reply:
Don.e